framework/tests/nontopdefault/NTDTest.java

```diff
@@ -0,0 +1,45 @@
+import tests.nontopdefault.qual.*;
+
+/* Hierarchy:
+ *    NTDTop (default for local variables, implicit upper bound, and receiver)
+ *    /     \
+ * NTDSide  NTDMiddle (default in hierarchy, default for exceptions and resource variables)
+ *   \       /
+ *   NTDBottom (default for implicit and explicit lower bounds, implicit for null literal and Void.class)
+ */
+
+// Because classes and interfaces are by default NTDMiddle, a single override is defined in the
```

Comment:
That is a note about the implementation of the type system. It's not relevant to this particular test.